cypress/integration/references-popup.js

```diff
@@ -0,0 +1,24 @@
+/// <reference types="Cypress" />
+
+import { ArticlePage } from '../page-objects/article-page'
+
+const articlePage = new ArticlePage()
+
+describe('references', () => {
+  beforeEach(() => {
+    cy.navigateToHomePage()
+  })
+
+  it.skip('check preview for references opens', () => {
+    goToCatArticle()
+    cy.downArrow()
+    cy.rightArrow()
+    cy.enter()
+    cy.get('.ref-title').should('be.visible').should('contain.text', 'Reference [10]')
```

Comment:
Hey @jpita, mine was opening one article preview in my local

![image](https://user-images.githubusercontent.com/2560096/93595676-7591a780-f9b8-11ea-9a98-e0f9a0c23e36.png)